### ISSUE TITLE
Add Apple TV device 

### ISSUE THREAD

**Issue by apereipa** [dmunozv04/iSponsorBlockTV#146]
Hello
I don't know if this is the most correct place, but I bought an Apple TV, and installed iSponsorBlockTV on my home assistant.
And I can't get it to work

in the logs appear this mensage:

[18:02:04] INFO: Starting...
No devices found, please add at least one device
Exiting in 10 seconds...

Can anyone help me? 
thanks

**Comment by dmunozv04** [comment]
As @bertybuttface ’s add on [instructions](https://github.com/bertybuttface/addons/blob/main/isponsorblocktv/DOCS.md#how-to-use) say, you need to generate a valid config file on another device and copy it to the device that’ll run the addon. It doesn’t automatically discover devices

**Comment by apereipa** [comment]
And I can use a windows pc to do that?

**Comment by dmunozv04** [comment]
Yeah, you should be able to either use docker, pip  or pipx to install iSponsorblockTV on your windows machine and copy the appropriate config.json to home assistant. Check the wiki for more details

**Comment by apereipa** [comment]
ok, It works!!

i install a docker desktop on windows, 
and then i run in my powershell this:

Remove-Item -Recurse -Force C:\tmp\config
New-Item -ItemType Directory -Force C:\tmp\config
docker run --rm -it -v C:\tmp\config:/app/data --net=host ghcr.io/dmunozv04/isponsorblocktv --setup-cli
Get-Content C:\tmp\config\config.json

**Comment by dmunozv04** [comment]
that's great to hear
